tools/UpdateModules.ps1

```diff
@@ -67,6 +67,21 @@ function Create-ModulePsm1
      $template = $template -replace "%DATE%", [string](Get-Date)
      $template = $template -replace "%IMPORTED-DEPENDENCIES%", $importedModules
 
+     if ($ModulePath -like "*Profile*")
+     {
+        $SpecialFolderPath = [Environment]::GetFolderPath("ApplicationData")
```

Comment:
This is exactly what I had in mind, but we want to make sure that any error in writing the file isn't bubbled up to the user.  The safest thing would be to wrap the file write in a try/catch